contract-tests/data-model/include/data_model/data_model.hpp

```diff
@@ -6,7 +6,6 @@
 #include "nlohmann/json.hpp"
 
 namespace nlohmann {
```

Comment:
Formatter inserted these newlines. The main changes are in `Config[Streaming|Polling]Params` adding the new filter key param.